docker/tpl/run.sh

```diff
@@ -0,0 +1,6 @@
+if [ ! -d "${DST_PATH}" ]; then
+  mkdir -p ${DST_PATH}
```

Comment:
I think this is not enough. If DEPTH is >1 you have to create sub dirs as well. Better to move this to L5 and use sub path of the src file to create dst sub path dirs.